### PR TITLE
Add support for schemes in build_ipa

### PR DESCRIPTION
**Pull request by younata**

### CODE REVIEW

**Comment by eugie** [comment]
Please update the example thrust.yml, the README and the CHANGELOG with documentation of this feature.  It might make sense to put this feature in Version 0.7.1.

**Comment by younata** [comment]
Updated the README and CHANGELOG. I didn't look particularly hard, but I didn't find an example thrust.yml outside of the one specified in the readme.

I thought of also updating the gemspec, but I think that should be included in a separate pull request? (By the way, the gemspec lists the version as 0.6.0)

File: .ruby-version [hunk 1]
```diff
@@ -1 +0,0 @@
-2.1.1
```
Comment (eugie) [inline .ruby-version]: Why did you delete the .ruby-version?

Comment (younata) [inline .ruby-version]: Because there's no point enforcing a ruby version at all. It was initially deleted because fuck no was I going to wait until ruby 2.1.1 was installed, and I just left it in after talking to people and basically getting a consensus that .ruby-version is pointless in public projects.

If you really want, I can add another commit which reintroduces the .ruby-version.

Comment (eugie) [inline .ruby-version]: I think that if you want to remove the .ruby-version then that should be in a separate pull request, since it is unrelated to the main change being introduced here.

**Comment by younata** [comment]
Amended the last commit to re-add .ruby-version.

**Comment by tjarratt** [comment]
Tests are passing, code looks good, and people (myself included) want to use this to build our projects that use schemes instead of targets. I'm going to merge this now and we can figure out later when v0.7.1 will be released.

**Comment by tjarratt** [comment]
Thank you @younata for submitting this, and thank you @eugie for taking the time to review it.
